### PR TITLE
BaSD/MS-47/Counselors redux

### PR DESCRIPTION
**Pull request by MicaQuarleri**
### **Important**:
First merge de server PR #36

### CODE REVIEW

File: src/Components/Layout/layout.module.css [hunk 1]
```diff
@@ -7,7 +7,7 @@
 .body {
   display: flex;
   flex-direction: row;
-  height: 85vh;
+  min-height: 160vh;
```
Comment (leimeterjoaquin) [inline src/Components/Layout/layout.module.css:10]: I think this affects everyone. if the minimum height is 160vh the layout is going to be bigger on every page. Also in the heroku deploy I see the old buttons. Did you pull development?

File: src/index.js [hunk 2]
```diff
@@ -4,12 +4,26 @@ import { BrowserRouter as Router } from 'react-router-dom';
 import Routes from './routes';
 import './reset.css';
 import './index.css';
+import { composeWithDevTools } from 'redux-devtools-extension';
```
Comment (leimeterjoaquin) [inline src/index.js:7]: This is already done in development.
- stand on your branch
- git fetch
- git merge origin/development
- resolve conflicts
- git add .
- git commit
- write :wq to exit the commit

**Review by leimeterjoaquin — CHANGES_REQUESTED** [review]
Rename PR to:
`BasD-MS-47: Counselors redux`

**Review by aquilesmariani — CHANGES_REQUESTED** [review]

File: src/Components/Layout/layout.module.css (hunk 1 above)
Comment (aquilesmariani) [reply src/Components/Layout/layout.module.css:10]: @MicaQuarleri remember to remove this `min-height` property

**Review by aquilesmariani — CHANGES_REQUESTED** [review]

**Review by aquilesmariani — APPROVED** [review]
Looks great!

**Review by leimeterjoaquin — APPROVED** [review]

**Pull request merged**